geth/account/interfaces.go

```diff
@@ -0,0 +1,45 @@
+package account
```

Comment:
I think it's antipattern to create interfaces for each new entity/structure without real need. Interfaces should be created by consumers.

Reply:
Would like once to discuss about it. There are two different ways:

- Type implementor also defines public interfaces which other use and which can be mocked
- Type implementor only codes the types but the user always uses an interface (so maybe N interfaces distributed over the using packages) to allow mocking

Reply:
@divan most of this interfaces narrows NodeManager responsibility and allows me to mock it.